src/rust/ide/lib/view/src/graph_editor.rs

```diff
@@ -970,16 +973,26 @@ pub struct GraphEditorModel {
 // === Public ===
 
 impl GraphEditorModel {
-    pub fn new<S:Into<Scene>>(scene:S, cursor:cursor::Cursor, network:&frp::Network) -> Self {
-        let scene          = scene.into();
+    pub fn new
+    ( world   : &World
+    , cursor  : cursor::Cursor
+    , network : &frp::Network
+    ) -> Self {
+        let scene          = world.scene().clone_ref();
         let logger         = Logger::new("GraphEditor");
         let display_object = display::object::Instance::new(&logger);
         let nodes          = Nodes::new(&logger);
 //        let visualizations = Stage::new(scene.clone_ref(), Logger::new("VisualisationCollection"));
         let edges          = default();
         let frp            = FrpInputs::new(network);
         let touch_state    = TouchState::new(network,&scene.mouse.frp);
-        Self {logger,display_object,scene,cursor,nodes,edges,touch_state,frp}//visualizations }
+        let project_name   = component::ProjectName::new(world);
+        scene.add_child(&display_object);
```

Comment:
why this change? Its not responsibility of graph editor model to add itself to scene!

Reply:
I was testing something (`TextField` was displayed - with some errors - even if it wasn't added as a child of a display object) and I forgot to remove this line.

Reply:
Always when testing add empty `// fixme` comments not to forget about it!